### PR TITLE
Document Amazon OneLink feature

### LINKED ISSUE #55: Add article about amazon one link support
Related to https://github.com/Pelican-Elegant/elegant/pull/248

### PR DESCRIPTION
**Pull request by iranzo**
Closes #55

### CODE REVIEW

File: content/Elegant - Pelican Theme/amazon-one-link.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+- [Canada](https://www.amazon.ca/dp/1449355730?s=books&ie=UTF8&qid=1546637046&sr=1-3&keywords=python&linkCode=sl1&linkId=34c3dec8f666e20197aa4674f8cbbe58&tag=iranzo0e-20)
+- [France](https://www.amazon.fr/dp/1449355730?s=books&ie=UTF8&qid=1546637046&sr=1-3&keywords=python&linkCode=sl1&linkId=34c3dec8f666e20197aa4674f8cbbe58&tag=iranzo03-21)
+- [Germany](https://www.amazon.de/dp/1449355730?s=books&ie=UTF8&qid=1546637046&sr=1-3&keywords=python&linkCode=sl1&linkId=34c3dec8f666e20197aa4674f8cbbe58&tag=iranzo06-21)
+- [Italy](https://www.amazon.it/dp/1449355730?s=books&ie=UTF8&qid=1546637046&sr=1-3&keywords=python&linkCode=sl1&linkId=34c3dec8f666e20197aa4674f8cbbe58&tag=iranzo04-21)
+- [Spain](https://www.amazon.es/dp/1449355730?s=books&ie=UTF8&qid=1546637046&sr=1-3&keywords=python&linkCode=sl1&linkId=34c3dec8f666e20197aa4674f8cbbe58&tag=iranzo-21)
+- [UK](https://www.amazon.co.uk/dp/1449355730?s=books&ie=UTF8&qid=1546637046&sr=1-3&keywords=python&linkCode=sl1&linkId=34c3dec8f666e20197aa4674f8cbbe58&tag=iranzo0f-21)
+
+Of course, if unset, no script load is instructed (nor present on the rendered pages) similar to what happens with piwik, google analytics, etc.
```
Comment (ashwinvis) [inline content/Elegant - Pelican Theme/amazon-one-link.md]: Just a suggestion, should we add a disclaimer that it is upto the content creator to appropriately seek consent from visitors while using the tools.

Comment (iranzo) [reply content/Elegant - Pelican Theme/amazon-one-link.md]: How will you phrase it?

Tried to indicate that is optional in line 53, but seems that it should be expanded.

Thanks!

Comment (ashwinvis) [reply content/Elegant - Pelican Theme/amazon-one-link.md]: It is quite clear what you mentioned. Also the MIT license we use makes us free of liability resulting from content creator's activities. Maybe you can simply mention:

> We assume that the you, as a content creator, are aware of [GDPR regulations](https://piwik.pro/blog/how-will-gdpr-affect-your-web-analytics-tracking/), and expect you to seek consent from your readers to use these plugins .

Comment (iranzo) [reply content/Elegant - Pelican Theme/amazon-one-link.md]: Done!

File: content/Elegant - Pelican Theme/amazon-one-link.md [hunk 2]
```diff
@@ -51,3 +51,5 @@ Gets translated by OneLink to:
 - [UK](https://www.amazon.co.uk/dp/1449355730?s=books&ie=UTF8&qid=1546637046&sr=1-3&keywords=python&linkCode=sl1&linkId=34c3dec8f666e20197aa4674f8cbbe58&tag=iranzo0f-21)
 
 Of course, if unset, no script load is instructed (nor present on the rendered pages) similar to what happens with piwik, google analytics, etc.
+
+We assume that the you, as a content creator, are aware of GDPR or other regulations, and expect you to seek consent from your readers to use these plugins .
```
Comment (ashwinvis) [inline content/Elegant - Pelican Theme/amazon-one-link.md]: That was quick! Please change "the you" -> "you". My bad :)

Comment (iranzo) [reply content/Elegant - Pelican Theme/amazon-one-link.md]: My fault for blindly copying :)

(done)

**Comment by iranzo** [comment]
Thanks!, let's wait for @talha131  to review or see of someone else wants to merge it

**Comment by talha131** [comment]
Thank you @ashwinvis for reviewing it.

**Pull request merged**